src/Xamarin.Android.NUnitLite/Gui/Instrumentations/TestSuiteInstrumentation.cs

```diff
@@ -103,47 +106,11 @@ public override void OnStart ()
 
 		string GetResultsPath ()
 		{
-			Java.IO.File resultsPathFile = GetExternalFilesDir ();
-			var usePathFile = resultsPathFile != null && resultsPathFile.Exists ();
-			var resultsPath = usePathFile
-				? resultsPathFile.AbsolutePath
-				: Path.Combine (Context.FilesDir.AbsolutePath, ".__override__");
-			if (!usePathFile && !Directory.Exists (resultsPath))
-				Directory.CreateDirectory (resultsPath);
-			return Path.Combine (resultsPath, "TestResults.xml");
-		}
-
-		Java.IO.File GetExternalFilesDir ()
-		{
-			if (((int)Build.VERSION.SdkInt) < 19)
-				return null;
-			string type = null;
-#if __ANDROID_19__
-			type = global::Android.OS.Environment.DirectoryDocuments;
-#else   // !__ANDROID_19__
-			type = global::Android.OS.Environment.DirectoryDownloads;
-#endif  // !__ANDROID_19__
-			return Context.GetExternalFilesDir (type);
-		}
-
-		// On some Android targets, the external storage directory is "emulated",
-		// in which case the paths used on-device by the application are *not*
-		// paths that can be used off-device with `adb pull`.
-		// For example, `Contxt.GetExternalFilesDir()` may return `/storage/emulated/foo`,
-		// but `adb pull /storage/emulated/foo` will *fail*; instead, we may need
-		// `adb pull /mnt/shell/emulated/foo`.
-		// The `$EMULATED_STORAGE_SOURCE` and `$EMULATED_STORAGE_TARGET` environment
-		// variables control the "on-device" (`$EMULATED_STORAGE_TARGET`) and
-		// "off-device" (`$EMULATED_STORAGE_SOURCE`) directory prefixes
-		string ToAdbPath (string path)
-		{
-			var source  = System.Environment.GetEnvironmentVariable ("EMULATED_STORAGE_SOURCE");
-			var target  = System.Environment.GetEnvironmentVariable ("EMULATED_STORAGE_TARGET");
-
-			if (!string.IsNullOrEmpty (source) && !string.IsNullOrEmpty (target) && path.StartsWith (target, StringComparison.Ordinal)) {
-				return path.Replace (target, source);
-			}
-			return path;
+			string pid = Guid.NewGuid ().ToString ();
+			string overridePath = Path.Combine (Context.FilesDir.AbsolutePath, ".__override__");
+			if (!Directory.Exists (overridePath))
+				Directory.CreateDirectory (overridePath);
```

Comment:
Should we put the test results in `.__override__`?

I'm just thinking if a Xamarin dev is using NUnitLite, they might have a better time getting access to the file if it was up one directory?

I think they would be able to use `run-as ... cat TestResults.xml`?